src/test/java/org/broadinstitute/consent/http/db/DacDAOTest.java

```diff
@@ -79,6 +80,26 @@ public void testFindAllDACUsersBySearchString_case2() {
         Assert.assertTrue(users.isEmpty());
     }
 
+    @Test
```

Comment:
Not your code, but on line 34, there is an extra semi-colon. Would love for those kinds of things to be cleaned up too.